### PR TITLE
[ACM-8480]: Adding TP statement for disconnected hcp docs

### PR DESCRIPTION
**Pull request by lahinson**
Related Jira: https://issues.redhat.com/browse/ACM-8480

### CODE REVIEW

**Comment by lahinson** [comment]
@zanetworker When you can (next week is fine), PTAL at this PR, which adds a statement that disconnected is TP for virtual platforms. Thanks!

File: clusters/hosted_control_planes/configure_hosted_disconnected.adoc [hunk 1]
```diff
@@ -6,7 +6,9 @@ In the context of hosted control planes, a disconnected environment is an {ocp-s
...
 == Disconnected environment architecture
 
-When you provision hosted control planes on bare metal, you use the Agent platform. The Agent platform and {mce-short} work together to enable disconnected deployments. The Agent platform uses the central infrastructure management service to add worker nodes to a hosted cluster. For an introduction to the central infrastructure management service, see xref:../cluster_lifecycle/cim_enable.adoc#enable-cim[Enabling the central infrastructure management service]. 
+When you provision hosted control planes on bare metal, you use the Agent platform. The Agent platform and {mce-short} work together to enable disconnected deployments. The Agent platform uses the central infrastructure management service to add worker nodes to a hosted cluster. For an introduction to the central infrastructure management service, see xref:../cluster_lifecycle/cim_enable.adoc#enable-cim[Enabling the central infrastructure management service].
+
+*Note:* You can deploy hosted control planes in a disconnected environment on bare-metal platforms. On a virtual platform, hosted control planes in a disconnected environment is available as a Technology Preview feature.
```
Comment (zanetworker) [inline clusters/hosted_control_planes/configure_hosted_disconnected.adoc]: Can we mention 'OpenShift Virtualization` as the virtual platform?

Comment (lahinson) [reply clusters/hosted_control_planes/configure_hosted_disconnected.adoc]: @zanetworker Yep - updated ✔️

**Review by zanetworker — APPROVED** [review]

**Review by swopebe — APPROVED** [review]

**Comment by lahinson** [comment]
FYI @YuLi517 this was the PR where I added the statement that dual-stack, disconnected is available as a TP feature.

**Pull request merged**